plugins/slick.state.js

```diff
@@ -51,6 +51,11 @@
       _store = options.storage,
       onStateChanged = new Slick.Event();
 
+    var userData = {
+      state: null,
+      current: null
```

Comment:
what is `current` for? Is that to keep reference to the previous data before it changes? I usually named them `previousX` but I guess `currentX` is also good enough except that it is previous in a few areas of the lib, you might want to check that

Reply:
I used `current` and `state` to represent the `userData` read from state and the its current set value (either read from stored state or set by user); therefore, those names.